fs/write_json.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+
+interface WriteJsonOption {
+  spaces?: number | string;
+  replacer?: (number | string)[] | null | Replacer;
+}
+
+type Replacer = (key: string, value: any) => any;
+
+/**
+ * Writes an object to a JSON file.
+ * @export
+ * @param {string} filePath
+ * @param {*} object
+ * @param {WriteJsonOption} [options]
+ * @returns {Promise<void>}
+ */
+export async function writeJson(
+  filePath: string,
+  object: any,
+  options?: WriteJsonOption
```

Comment:
[Style guide](https://deno.land/style_guide.html) suggest that `options` should have default value instead of being optional. 

CC @piscisaureus maybe it's worth adding to style guide that `options` should have default value over being optional?

Reply:
@bartlomieju yes. my bad